### PR TITLE
Feature post api intergration

### PR DESCRIPTION
**Pull request by purisaurabh**
- This pr contains the all post api integration

### CODE REVIEW

File: src/api/store/store.jsx [hunk 1]
```diff
@@ -17,19 +17,27 @@ const preloadedState = {
 const rootReducer = combineReducers({
   [loginApi.reducerPath]: loginApi.reducer,
```
Comment (Payal13) [inline src/api/store/store.jsx:18]: please check constants of reducer path, they are different from what you have used

File: src/components/Builder/Index.jsx [hunk 2]
```diff
@@ -17,64 +18,64 @@ const createPanes = (profileData, disableTabs) => [
   {
     key: "basic-info",
     label: <b>Basic Info</b>,
```
Comment (Payal13) [inline src/components/Builder/Index.jsx]: Add constants for these labels as well.  and remove <b>
use its default styling

File: src/components/Profile/List/index.jsx [hunk 3]
```diff
@@ -181,23 +179,22 @@ const ListProfiles = () => {
...
-          columns={columns}
-          dataSource={data?.profiles}
-          className={styles.table}
-          bordered={true}
-          loading={isFetching}
-        />
-      </div>
+      <Typography.Title level={1} className={styles.profile_header}>
+        Profiles
+        <Link to={`/profile-builder`}>
```
Comment (Payal13) [inline src/components/Profile/List/index.jsx]: Use const route

File: src/services/axios.js [hunk 4]
```diff
@@ -28,7 +29,10 @@ axiosInstance.interceptors.response.use(
     return response;
   },
   (error) => {
-    // Handle response errors here
+    console.log("error in response interceptor", error);
```
Comment (Payal13) [inline src/services/axios.js]: remove console

File: src/components/Profile/List/index.jsx (hunk 3 above)
Comment (purisaurabh) [reply src/components/Profile/List/index.jsx]: done

File: src/services/axios.js (hunk 4 above)
Comment (purisaurabh) [reply src/services/axios.js]: done

File: src/api/store/store.jsx (hunk 1 above)
Comment (purisaurabh) [reply src/api/store/store.jsx:18]: the above is format which is given by rtk query

File: src/components/Builder/Index.jsx (hunk 2 above)
Comment (purisaurabh) [reply src/components/Builder/Index.jsx]: done

**Review by Payal13 — APPROVED** [review]

**Pull request merged**